files/fr/learn/css/building_blocks/backgrounds_and_borders/index.html

```diff
@@ -1,6 +1,15 @@
 ---
 title: Arrière-plans et bordures
 slug: Learn/CSS/Building_blocks/Backgrounds_and_borders
+tags:
+  - Background
+  - Beginner
+  - CSS
+  - Image
+  - Learn
+  - Position
+  - borders
+  - color
```

Comment:
Tags are never used or read in the translated content. No point adding them.

Reply:
Ok, I finish to check all pages and I remove the tags of this page ^^